broadinstitute/pearl
# JN-320 enabling offline population

This was useful to me during my commute this morning.  Currently, to populate, you need to get an Azure token.  Now, the default is just to use a generated "fake" token.  This has the added benefit of adding an extra safeguard against accidentally populating against demo or production servers.

TO TEST:
1. turn of wifi
2. Run `./scripts/populate_portal.sh ourhealth`
3. confirm success
